src/odemis/acq/stitching/test/registrar_test.py

```diff
@@ -0,0 +1,533 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 26 Jul 2017
+
+@author: Éric Piel, Philip Winkler
+
+Copyright © 2017 Éric Piel, Philip Winkler, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms 
+of the GNU General Public License version 2 as published by the Free Software 
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR 
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with 
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+
+from __future__ import division
+import logging
+from odemis import model
+import numpy
+import unittest
+import random
+import copy
+import os
+
+from odemis.acq.stitching import IdentityRegistrar, ShiftRegistrar, decompose_image
+from odemis.dataio import tiff
+
+logging.getLogger().setLevel(logging.DEBUG)
+
+# Find path for test images
+path = os.path.relpath("acq/align/test/images", "acq/stitching/test")
+imgs = [path + "/Slice69_stretched.tif"]
```

Comment:
Rename `imgs` to IMGS. Don't use `Slice69_stretched.tif`, use `songbird-sim-ccd.h5` and `songbird-sim-sem.h5`. 
Instead of `path`, use:
`IMG_PATH = os.path.dirname(odemis.__file__) + "driver/"`

Reply:
The songbird images don't have a lot of content, especially the ccd image. Some regions are entirely black, so the stitching algorithm is not going to work very well. I use the songbird-sem now for the registrar test, but I had to increase the tolerated error threshold. Songbird-ccd still fails the tests.

Reply:
You can also try simsem-fake-output.h5 and sparc-etd-gold-balls.h5 (though it has not much high frequency so it might not work so well).